.pipelines/templates/test-docs.yml

```diff
@@ -0,0 +1,76 @@
+trigger: none
+pr: none
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.x'

```

Comment:
wouldn't you want to run this pipeline in multiple versions?

Reply:
good point, I'll add a metrix for py versions

Reply:
does it change the result or the code? why not test for all supported version?

Reply:
There shoundt be an impact from the python version, and since it just a matter of validating the samples against the latest version of presidio, testing on one python version should be sufficient.